src/components/select/select.ts

```diff
@@ -151,6 +151,7 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
 
   _multi: boolean = false;
   _options: QueryList<Option>;
+  private _overlay: ActionSheet | Alert | Popover;
```

Comment:
I am not sure, we are not using private (probably ability for unit testing), let's drop the private for the moment.

We could re-evaluate it in the future (a different PR)